cmd/snap-confine/spread-tests/main/network-namespace-sharing/task.yaml

```diff
@@ -0,0 +1,34 @@
+summary: The /run/netns directory propagates events outwards
```

Comment:
This makes sense (obviously, sorry for not realizing this earlier).

We can now run spread tests as snap-confine is merged at the packaging level in master so we can revisit this.